lib/error_tracker/schemas/error.ex

```diff
@@ -12,7 +12,13 @@ defmodule ErrorTracker.Error do
 
   use Ecto.Schema
 
-  @type t :: %__MODULE__{}
+  @type t :: %__MODULE__{
+    kind: string(),
+    reason: string(),
+    source_line: string(),
+    source_function: string(),
+    status: :resolved | :unresolved
+  }
```

Comment:
Why not adding other fields? 

I can think of `last_occurrence_at` or `muted` to be useful for integrations using telemetry or even for the ignorer. `fingerprint` and timestamps may not be so useful but I would add all of them.

Reply:
Thank you for the feedback @odarriba  -- honestly, I didn't see muted, and it wasn't obvious whether `fingerprint` or `last_occurrence_at` were intended to be user-facing, and I didn't want to